store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -125,4 +101,14 @@ public String call() throws Exception {
         return sanitize();
     }
 
+    protected Set<String> getSkipTagsUnclosedCdata() {
+        Set<String> zimbraSkipTagsWithUnclosedCdata = new HashSet<>();
+        try {
+            zimbraSkipTagsWithUnclosedCdata = new HashSet<>(
+                    Arrays.asList(LC.zimbra_skip_tags_with_unclosed_cdata.value().toString().split(",")));
+        } catch (Exception ex) {
+            ZimbraLog.mailbox.error("Issue getting local config value ", ex);
```

Comment:
add more detail to error about getting which LC value failed

Reply:
added LC name

Reply:
done